packages/story-editor/src/components/fontPicker/index.js

```diff
@@ -26,50 +26,31 @@ import {
 import PropTypes from 'prop-types';
 import { __ } from '@googleforcreators/i18n';
 import { Datalist } from '@googleforcreators/design-system';
-import { stripHTML } from '@googleforcreators/dom';
 
 /**
  * Internal dependencies
  */
-import { useFont } from '../../../../app/font';
-import { MULTIPLE_DISPLAY_VALUE, MULTIPLE_VALUE } from '../../../../constants';
-import { getCommonValue } from '../../shared';
-import useRichTextFormatting from './useRichTextFormatting';
-import getClosestFontWeight from './getClosestFontWeight';
+import { useFont } from '../../app';
+import { MULTIPLE_DISPLAY_VALUE, MULTIPLE_VALUE } from '../../constants';
 
 const FontPicker = forwardRef(function FontPicker(
-  { selectedElements, pushUpdate, highlightStylesOverride },
+  { onChange, currentValue, highlightStylesOverride },
```

Comment:
Per 

https://github.com/GoogleForCreators/web-stories-wp/pull/10808/files#diff-a802a3ecd6f447cf10aa58abf9e7f3d4e7ec9b458ccba02a3b39ad2a7df4f48fR190

I added a `z-Index` prop as well (if there's a better way to implement this let me know ... also should they land in this PR or in #10808 )

Commit:
https://github.com/GoogleForCreators/web-stories-wp/pull/10808/commits/8bd605b011001c4cb1573af393e5dc525b9919f1#diff-a802a3ecd6f447cf10aa58abf9e7f3d4e7ec9b458ccba02a3b39ad2a7df4f48fR190


Issue noted here:
https://github.com/GoogleForCreators/web-stories-wp/issues/10014#issuecomment-1057569298

Reply:
In #10808 should be fine